servers/display_server.h

```diff
@@ -189,6 +191,7 @@ class DisplayServer : public Object {
 		return scale;
 	}
 	virtual bool screen_is_touchscreen(int p_screen = SCREEN_OF_MAIN_WINDOW) const;
+	virtual float screen_get_refresh_rate(int p_screen = SCREEN_OF_MAIN_WINDOW) const = 0;
```

Comment:
To match the order in which it was listed in implementation classes:
```suggestion
	virtual float screen_get_refresh_rate(int p_screen = SCREEN_OF_MAIN_WINDOW) const = 0;
	virtual bool screen_is_touchscreen(int p_screen = SCREEN_OF_MAIN_WINDOW) const;
```

Reply:
Thanks, fixed!